MediaAdminBundle/Resources/config/form.yml

```diff
@@ -14,11 +14,26 @@ parameters:
                 icon: false
         use_callback_tinymce_init: true
 
+    open_orchestra_media_admin.allowed_mime_type:
```

Comment:
@nicolasThal : When we defined that task, we said the opposite as for now there's now flexible mechanism on the upload process to add custom strategies. So for now, integrators should not be able to touch those settings.
It will be possible later, when we'll have done a refacto on that process,